tools/ci_build/github/azure-pipelines/nuget/templates/cpu-nocontribops-arm64.yml

```diff
@@ -6,6 +6,7 @@
 parameters:
   DoEsrp: 'false'
 
+jobs:  
```

Comment:
For some reason, I am able to manually trigger a job only after adding this line. I don't quite know how it was working previously without this.